templates/cisco_nxos_show_interface_status.template

```diff
@@ -3,7 +3,7 @@ Value NAME (.*?)
 Value STATUS (\S+)
 Value VLAN (\d+|routed|trunk)
 Value DUPLEX (\S+)
-Value SPEED (\d+|auto)
+Value SPEED (\d+G?|auto)
```

Comment:
```suggestion
Value SPEED (\S+)
```

Reply:
Commited this but also had to change the regex for `TYPE` to pass all tests